### DISCUSSION TITLE
Latency issues on Windows

### DISCUSSION THREAD

**Discussion by rubenwilson1** [BearWare/TeamTalk5#1106] [General]
good evening 
I would like to make a query, 
in the latest versions the following problems are occurring: 
We notice a lot of delay when talking between several users, is it possible to correct it? because we have the buffer to the minimum and it still has a delay, (latency) 
2, I would like to know if it is possible to apply the dark mode to the client, there is a person who uses it and tells me that he does not know how to put the teamtalk client in dark mode and he needs it because he does not see the interface well, 
in addition to that in windows 11 the client closes alone and we do not know why 
a greeting

**Comment by bear101** [comment]
Please state platform, client application and selected sound system

**Comment by rubenwilson1** [comment]
the server I connect to is linux, encrypted,
the client is windows, and right now I use directsound but I have tried with windows default as well

**Comment by bear101** [comment]
Windows Audio Session gives the lowest latency on Windows.

**Comment by bear101** [comment]
Also using Windows Audio Session, try enabling echo cancellation because this will merge all voice stream into a single stream (what was previously called Duplex-mode). This configuration uses more CPU but gives lower latency.

**Comment by rubenwilson1** [comment]
I'm going to try it perfect, and what can you tell me about the dark mode issue? is it possible to put the interface in dark mode? and how would I? or are you working on something about it?

**Comment by bear101** [comment]
Are you able to download artifacts here: https://github.com/BearWare/TeamTalk5/actions/runs/1424801299
Then you should have dark-mode enabled if your Windows is configured for Dark-mode. Note that the builds from GitHub do not contain libvpx and WebRTC for noise reduction and echo cancellation.
